charts/chainlink-cluster/devspace.yaml

```diff
@@ -25,47 +25,14 @@ pipelines:
       start_dev app                # 3. Start dev mode "app" (see "dev" section)
   deploy:
     run: |-
-      set -o pipefail
-      echo "Removing .devspace cache!"
-      rm -rf .devspace/ || true
-      registry_id=$(echo "$DEVSPACE_IMAGE" | cut -d'.' -f1)
-
-      # Login into registry
-      echo "Authorizing into ECR registry"
-      aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin ${registry_id}.dkr.ecr.us-west-2.amazonaws.com
-
       run_dependencies --all
       ensure_pull_secrets --all
-      
       build_images --all
       
       kubectl label namespace ${DEVSPACE_NAMESPACE} cleanup.kyverno.io/ttl=${NS_TTL} || true
       kubectl label namespace/${DEVSPACE_NAMESPACE} network=crib || true
 
-      tag=$(git rev-parse --short HEAD)
-      image=${DEVSPACE_IMAGE}:$tag
-      echo "Checking tag: '$tag'"
-      repository_name="chainlink-devspace"
-      desired_tag=$tag
-
-      # Check if the desired tag is present in the repository
-      image_list=$(aws ecr list-images --repository-name "$repository_name")
-      tag_exists=$(echo "$image_list" | jq -e '.imageIds[] | select(.imageTag == "'"${desired_tag}"'")' >/dev/null && echo true || echo false)
-
-      # Check the value of the tag_exists variable
-      if [ "$tag_exists" = "true" ]; then
-        echo "Image tag '$tag' found."
-      else
-        echo "Image tag '$tag' not found. Please build the image using 'devspace deploy'"
-        exit 1
-      fi
-      create_deployments app \
```

Comment:
If we're removing this here, don't we have to set it in the `app` section? These values make their way to the helm chart.

Reply:
devspace automatically does image tag replacement https://www.devspace.sh/docs/4.x/configuration/images/basics#6-tag-image